docs/migration-script-omega1.sql

```diff
@@ -0,0 +1,166 @@
+-- MySQL Script generated by MySQL Workbench
```

Comment:
A migration implies that no data will be lost, but we are starting off by dropping the databaase. Can we instead replace Pantry.sql with this file?